.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
```

Comment:
Could we rename this CI so we have more specific names on the PRs?

Reply:
Thanks for your review Tim!

Yeah we might come up with a better name in the future. The reason why I’ve picked ‘CI’ for now is to end up with ‘CI / Linting’, ‘CI / Playwright tests’ and so on. All these jobs can be stored in one yaml file, which is what we do in the Blockprotocol repo too.

Once we’ve finished configuring this monorepo, we can step back and take another look at various workflow names. I’m sure we’ll be in a better position to configure all yamls properly! Perhaps, if we add Turborepo, some questions will even self-resolve 🙂